types/voilab-pdf-table/index.d.ts

```diff
@@ -0,0 +1,221 @@
+// Type definitions for voilab-pdf-table 0.5
+// Project: https://github.com/voilab/voilab-pdf-table
+// Definitions by: Daniel Sousa <https://github.com/danielb7390>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="pdfkit" />
+
+export = VoilabPdfTable;
+export as namespace VoilabPdfTable;
```

Comment:
```suggestion
export as namespace VoilabPdfTable;
```
do you need those global script features? I cannot see this package used in browser via regular script tag.

Reply:
Indeed i don't..i misunderstood the meaning of  `export as namespace`.

Reply:
no worries, thx!